.github/workflows/benchmark.yml

```diff
@@ -112,6 +110,9 @@ jobs:
           ref: ${{ github.sha }}
           submodules: 'recursive'
 
+      - name: "Install dependencies"
+        run: source velox/scripts/setup-ubuntu.sh
```

Comment:
Running this twice on PRs shouldn't affect anything right?

Reply:
No it will just run the apt installation and non of the functions that will build stuff from source. 

And apt skips installed packages :)